### ISSUE TITLE
Fix code component layout for mobile devices

### ISSUE THREAD

**Issue by KutiBond** [mckaywrigley/chatbot-ui#183]
The code component on this project is not optimized for mobile devices. The layout issue makes it difficult for users to view and interact with the code on smaller screens.

To fix this issue, you can update the ChatMessage.tsx:
```
<div className="relative flex w-[calc(100%-50px)] flex-col gap-1 md:gap-3 lg:w-[calc(100%-115px)]">
  <div className="flex flex-grow flex-col gap-3">
    <div className="min-h-[20px] flex flex-col items-start gap-4">
      /// markdown render
    </div>
  </div>
</div>
```

**Comment by srsholmes** [comment]
If you have a fix for this issue then its probably worth doing a PR 🙂

**Comment by joestaxk** [comment]
i should do that

**Comment by SyedMuzamilM** [comment]
It is already fixed with this PR #258 
You can close the issue or let us know if there are other issue in the code component
